### ISSUE TITLE
CPU_COUNT

### ISSUE THREAD

**Issue by notestaff** [conda-forge/conda-forge-build-setup-feedstock#100]
CPU_COUNT here is set to 2; but current conda-smithy on azure seems to set max jobs to 8? https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/templates/azure-pipelines-linux.yml.tmpl#L11 @jakirkham

**Comment by scopatz** [comment]
It depends on the platform

**Comment by jakirkham** [comment]
So you are wanting to change [this line]( https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/a82d5374eb8f5106b0134b2c64c4fafb4318eb09/recipe/run_conda_forge_build_setup_linux#L5 )? Seems reasonable. Though that value is correct for CircleCI (just incorrect for Azure). So we will need to detect which is running and set it accordingly.

cc-ing @mariusvniekerk (in case he has thoughts here)

**Comment by notestaff** [comment]
right; also, ideally, when testing locally with docker, the local machine's CPU count would be used.
